examples/train.py

```diff
@@ -174,89 +180,215 @@ def setup_exps_rllib(flow_params,
     return alg_run, gym_name, config
 
 
-if __name__ == "__main__":
-    flags = parse_args(sys.argv[1:])
+def train_rllib(submodule, flags):
+    """Train policies using the PPO algorithm in RLlib."""
+    flow_params = submodule.flow_params
+    n_cpus = submodule.N_CPUS
+    n_rollouts = submodule.N_ROLLOUTS
+    policy_graphs = getattr(submodule, "POLICY_GRAPHS", None)
+    policy_mapping_fn = getattr(submodule, "policy_mapping_fn", None)
+    policies_to_train = getattr(submodule, "policies_to_train", None)
+
+    alg_run, gym_name, config = setup_exps_rllib(
+        flow_params, n_cpus, n_rollouts,
+        policy_graphs, policy_mapping_fn, policies_to_train)
+
+    ray.init(num_cpus=n_cpus + 1, object_store_memory=200 * 1024 * 1024)
+    exp_config = {
+        "run": alg_run,
+        "env": gym_name,
+        "config": {
+            **config
+        },
+        "checkpoint_freq": 20,
+        "checkpoint_at_end": True,
+        "max_failures": 999,
+        "stop": {
+            "training_iteration": flags.num_steps,
+        },
+    }
+
+    if flags.checkpoint_path is not None:
+        exp_config['restore'] = flags.checkpoint_path
+    run_experiments({flow_params["exp_tag"]: exp_config})
+
+
+def train_h_baselines(submodule, flags, args, multiagent):
+    """Train policies using SAC and TD3 with h-baselines."""
+    from hbaselines.algorithms import OffPolicyRLAlgorithm
+    from hbaselines.utils.train import parse_options, get_hyperparameters
+
+    # Get the command-line arguments that are relevant here
+    args = parse_options(description="", example_usage="", args=args)
+    print(args.env_name)
+
+    # the base directory that the logged data will be stored in
+    if multiagent:
+        base_dir = "training_data/multi-fcnet"
+    else:
+        base_dir = "training_data/fcnet"
```

Comment:
feel like this should go in the home directory rather than inside of flow. Amenable to different opinions